### PR TITLE
Updates for releases 1.0.2s, 1.1.0k and 1.1.1c

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: news/newsflash.txt [hunk 1]
```diff
@@ -4,6 +4,9 @@
 # Format is two fields, colon-separated; the first line is the column
 # headings.  URL paths must all be absolute.
 Date: Item
+26-Feb-2019: OpenSSL 1.1.1c is now available, including bug and security fixes
+26-Feb-2019: OpenSSL 1.1.0k is now available, including bug and security fixes
+26-Feb-2019: OpenSSL 1.0.2s is now available, including bug fixes
```
Comment (mattcaswell) [inline news/newsflash.txt]: Date is wrong

**Review by mattcaswell — CHANGES_REQUESTED** [review]

File: news/newsflash.txt (hunk 1 above)
Comment (mspncp) [inline news/newsflash.txt]: 28-May-2019 !!

Comment (levitte) [reply news/newsflash.txt]: Fixed

Comment (levitte) [reply news/newsflash.txt]: One at a time, please :wink:

**Review by mattcaswell — APPROVED** [review]

**Comment by levitte** [comment]
Merged.

File: news/newsflash.txt (hunk 1 above)
Comment (mspncp) [reply news/newsflash.txt]: I learned that with you I need to be very fast, Monsieur Le Vite!  ;-)

**Pull request closed without merging**